evalml/tests/component_tests/test_time_series_regularizer.py

```diff
@@ -60,16 +76,47 @@ def assert_features_and_length_equal(
             X["dates"] == each_date,
             list(set(X.columns) - {"dates"}),
         ].iloc[0]
-        outout_feat = non_nan_X.loc[
+        output_feat = non_nan_X.loc[
             non_nan_X["dates"] == each_date,
             list(set(non_nan_X.columns) - {"dates"}),
         ].iloc[0]
         pd.testing.assert_series_equal(
             pd.Series(input_feat.values),
-            pd.Series(outout_feat.values),
+            pd.Series(output_feat.values),
+            check_dtype=check_dtype,
         )
 
 
+def check(is_multiseries, X, y, X_output, y_output, error_dict):
```

Comment:
Can the function name be slightly more descriptive? It's not immediately obvious *what* is being checked here